acapy/start.sh

```diff
@@ -18,4 +21,4 @@ aca-py start \
     --wallet-storage-type postgres_storage \
     --admin 0.0.0.0 ${MEDIATOR_AGENT_HTTP_ADMIN_PORT} \
     --admin-api-key ${MEDIATOR_AGENT_ADMIN_API_KEY} \
-    --endpoint ${MEDIATOR_URL} wss://${MEDIATOR_URL#*://*}
+    --endpoint ${MEDIATOR_URL} ws${MEDIATOR_URL/http/}
```

Comment:
Why use `https` but not `wss`?

Reply:
The replace statement here works with http or https to ws or wss, as the http part will be replaced, but the s will carry over if it is there.